src/view/reviewManager.ts

```diff
@@ -704,6 +704,25 @@ export class ReviewManager implements vscode.DecorationProvider {
 		};
 	}
 
+	private async getPullRequestNameSetting(): Promise<PullRequestNameSource | undefined> {
+		const method = vscode.workspace.getConfiguration('githubPullRequests').get<PullRequestNameSource>('pullRequestName', 'ask');
+
+		if(method == "ask") {
```

Comment:
Looks like there are a couple of tslint warnings in this file. They're printed as part of the build, but at the top, so easy to miss. Using the tslint extension should show them in the editor